dotenv-linter/tests/common/test_dir.rs

```diff
@@ -14,20 +15,32 @@ use std::str::from_utf8;
 /// Use to test commands in temporary directories
 pub struct TestDir {
     current_dir: TempDir,
+    envs: Option<HashMap<String, String>>,
 }
 
 impl TestDir {
     /// Generate a new TestDir
     pub fn new() -> Self {
         Self {
             current_dir: tempdir().expect("create testdir"),
+            envs: None,
+        }
+    }
+
+    // Only used in tests
+    #[allow(dead_code)]
+    pub fn with_envs(envs: Option<HashMap<String, String>>) -> Self {
```

Comment:
There is no need to keep this method with optional param, it's supposed to be always with envs, otherwise `new` exists

Reply:
Updated here! https://github.com/dotenv-linter/dotenv-linter/pull/625/commits/c64b3667d04861528bf4a93e506ffca117fb4e07